website/client/src/components/auth/login.vue

```diff
@@ -0,0 +1,244 @@
+  >
+    <div class="text-center">
+      <div class="header">
+        <h2 v-once>
+          {{ $t('emailNewPass') }}
+        </h2>
+        <p v-once>
+          {{ $t('forgotPasswordSteps') }}
+        </p>
+      </div>
+    </div>
+    <div class="form-group row text-center">
+      <label
+        v-once
+        for="usernameInput"
+      >{{ $t('email') }}</label>
+      <input
+        id="usernameInput"
+        v-model="username"
+        class="form-control"
+        type="text"
+        :placeholder="$t('emailPlaceholder')"
+      >
+    </div>
+    <div class="text-center">
+      <div
+        v-once
+        class="btn btn-info"
+        @click="forgotPasswordLink()"
+      >
+        {{ $t('sendLink') }}
+      </div>
+    </div>
+  </form>
+  <form
+    v-else
+    id="login-form"
+    @submit.prevent.stop="login"
+  >
+    <div
+      class="form-group"
+    >
+      <label
+        v-once
+        for="usernameInput"
+      >{{ $t('emailOrUsername') }}</label>
+      <input
+        id="usernameInput"
+        v-model="username"
+        class="form-control"
+        type="text"
+        :placeholder="$t('emailOrUsername')"
+      >
+    </div>
+    <div class="form-group">
+      <label
+        v-once
+        for="passwordInput"
+      >{{ $t('password') }}</label>
+      <a
+        v-once
+        class="float-right forgot-password"
+        style="color: rgb(189, 168, 255) !important;"
+        @click="handleForgotPasswordChange"
+      >{{ $t('forgotPassword') }}</a>
+      <input
+        id="passwordInput"
+        v-model="password"
+        class="form-control"
+        type="password"
+        :placeholder="$t('password')"
+      >
+    </div>
+    <div class="text-center">
+      <button
+        v-once
+        type="submit"
+        class="btn btn-info"
+      >
+        {{ $t('login') }}
+      </button>
+      <div class="toggle-links">
+        <router-link
+          :to="{name: 'register'}"
+          exact="exact"
+        >
+          <a
+            v-once
+            class="toggle-link"
+            v-html="$t('dontHaveAccountSignup')"
+          ></a>
+        </router-link>
+      </div>
+    </div>
+  </form>
+</template>
+
+<style lang="scss" scoped>
+  @import '~@/assets/scss/colors.scss';
+  #login-form, #forgot-form {
+    ::-webkit-input-placeholder { /* Chrome/Opera/Safari */
+      color: $purple-400;
+    }
+    ::-moz-placeholder { /* Firefox 19+ */
+      color: $purple-400;
+    }
+    :-ms-input-placeholder { /* IE 10+ */
+      color: $purple-400;
+    }
+    :-moz-placeholder { /* Firefox 18- */
+      color: $purple-400;
+    }
+    ::placeholder { //  Standard browsers
+      color: $purple-400;
+  }
+
+    margin: 0 auto;
+    width: 40em;
+    padding-top: 0em !important;
+    padding-bottom: 0em !important;
+    position: relative;
+    z-index: 1;
+
+    .header {
+      h2 {
+        color: $white;
+      }
+
+      color: $white;
+    }
+
+    .gryphon {
```

Comment:
I can't find some of these classes and ids used in this component (`.gryphon`, `.habitica-logo`, `#confirmPasswordInput`)